tags/Algolia/Search Insights/gtm-algolia-search-insights-custom-template.tpl

```diff
@@ -0,0 +1,810 @@
+        aa(data.eventType, algoliaEventParams);
+        break;
+        // clickedObjectIDsAfterSearch  
+    case "clickedObjectIDsAfterSearch":
+    	if (checkIfInitialized() === false) {
+            log("GTM-DEBUG :: ALGOLIA >> You need to call the \"init\" event first");
+            break;          
+        }
+
+        log("GTM-DEBUG :: ALGOLIA >> Event: ", [data.eventType, algoliaEventParams]);
+        aa(data.eventType, algoliaEventParams);
+        break;
+        // clickedObjectIDs  
+    case "clickedObjectIDs":
+    	if (checkIfInitialized() === false) {
+            log("GTM-DEBUG :: ALGOLIA >> You need to call the \"init\" event first");
+            break;          
+        }
+        log("GTM-DEBUG :: ALGOLIA >> Event: ", [data.eventType, algoliaEventParams]);
+        aa(data.eventType, algoliaEventParams);
+        break;
+        // clickedFilters  
+    case "clickedFilters":
+    	if (checkIfInitialized() === false) {
+            log("GTM-DEBUG :: ALGOLIA >> You need to call the \"init\" event first");
+            break;          
+        }
+        log("GTM-DEBUG :: ALGOLIA >> Event: ", [data.eventType, algoliaEventParams]);
+        aa(data.eventType, algoliaEventParams);
+        break;
+        // convertedObjectIDsAfterSearch  
+    case "convertedObjectIDsAfterSearch":
+    	if (checkIfInitialized() === false) {
+            log("GTM-DEBUG :: ALGOLIA >> You need to call the \"init\" event first");
+            break;          
+        }
+        log("GTM-DEBUG :: ALGOLIA >> Event: ", [data.eventType, algoliaEventParams]);
+        aa(data.eventType, algoliaEventParams);
+        break;
+        // convertedObjectIDs  
+    case "convertedObjectIDs":
+    	if (checkIfInitialized() === false) {
+            log("GTM-DEBUG :: ALGOLIA >> You need to call the \"init\" event first");
+            break;          
+        }
+        log("GTM-DEBUG :: ALGOLIA >> Event: ", [data.eventType, algoliaEventParams]);
+        aa(data.eventType, algoliaEventParams);
+        break;
+        // convertedFilters  
+    case "convertedFilters":
+    	if (checkIfInitialized() === false) {
+            log("GTM-DEBUG :: ALGOLIA >> You need to call the \"init\" event first");
+            break;          
+        }
+        log("GTM-DEBUG :: ALGOLIA >> Event: ", [data.eventType, algoliaEventParams]);
+        aa(data.eventType, algoliaEventParams);
+        break;
+        // viewedFilters  
+    case "viewedFilters":
+    	if (checkIfInitialized() === false) {
+            log("GTM-DEBUG :: ALGOLIA >> You need to call the \"init\" event first");
+            break;          
+        }
+        log("GTM-DEBUG :: ALGOLIA >> Event: ", [data.eventType, algoliaEventParams]);
+        aa(data.eventType, algoliaEventParams);
+        break;    
+    default:
+        break;
+}
+// Call data.gtmOnSuccess when the tag is finished.
+data.gtmOnSuccess();
+
```

Comment:
You should remove this `data.gtmOnSuccess()` - otherwise the tag will always signal success, even if the script load fails.

Reply:
Done, also removed 2 lines where userToken was holding a fixed value ( from some testings I was doing )